app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"], person["age"]) for person in people]
+    for person in people:
+        if "wife" in person:
+            if person["wife"] is not None:
+                Person.people[person["name"]].wife = (
+                    Person.people[person["wife"]])
+        elif "husband" in person:
+            if person["husband"] is not None:
```

Comment:
Similar to the previous comment, you can combine these nested 'if' statements using the 'and' logical operator to make the code cleaner and more readable.